cmds/admin_server/server/server.go

```diff
@@ -1,29 +1,69 @@
 package server
 
 import (
+	"errors"
 	"fmt"
 	"net/http"
+	"os"
 
 	"github.com/gin-gonic/gin"
+	"github.com/linuxboot/contest/cmds/admin_server/storage"
 	"github.com/linuxboot/contest/pkg/xcontext"
 )
 
-func initRouter() *gin.Engine {
-	r := gin.New()
+type RouteHandler struct {
+	storage storage.Storage
+}
+
+// status is a simple endpoint to check if the serves is alive
+func (r *RouteHandler) status(c *gin.Context) {
+	c.JSON(http.StatusOK, gin.H{"status": "live"})
+}
 
+// addLog inserts a new log entry inside the database
+func (r *RouteHandler) addLog(c *gin.Context) {
+	var logBody storage.Log
```

Comment:
the type is Log, the method is called StoreLog, so why logBody
```suggestion
	var log storage.Log
```